tools/cli/internal/changelog/changelog.go

```diff
@@ -229,6 +229,12 @@ func NewEntriesBetweenRevisionVersions(revisionPath, exceptionFilePath string) (
 			if apiversion.IsPreviewStabilityLevel(fromVersion) || apiversion.IsPreviewStabilityLevel(toVersion) {
 				continue
 			}
+
+			// Skip upcoming preview version. It will be included in CLOUDP-315486
+			if apiversion.IsUpcomingStabilityLevel(fromVersion) || apiversion.IsUpcomingStabilityLevel(toVersion) {
```

Comment:
maybe we only add it once it turns into the version, no? so we would never add upcoming to the changelog?